nextcord/ext/commands/cog.py

```diff
@@ -191,21 +124,18 @@ def qualified_name(cls) -> str:
         return cls.__cog_name__
 
 
-class Cog(ClientCog, metaclass=CogMeta):
-    """The base class that all cogs must inherit from.
+class CommandCog(Cog, metaclass=CommandCogMeta):
+    """A base class that adds support for prefixed commands.
 
-    A cog is a collection of commands, listeners, and optional state to
-    help group commands together. More information on them can be found on
-    the :ref:`ext_commands_cogs` page.
-
-    When inheriting from this class, the options shown in :class:`CogMeta`
-    are equally valid here.
+    This class inherits from :class:`Cog` and uses the :class:`CommandCogMeta`
+    metaclass. More information on them can be found on the
+    :ref:`ext_commands_cogs` page.
     """
 
     __cog_name__: ClassVar[str]
     __cog_settings__: ClassVar[Dict[str, Any]]
     __cog_commands__: ClassVar[List[Command]]
```

Comment:
```suggestion
    __cog_name__: str
    __cog_settings__: Dict[str, Any]
    __cog_commands__: List[Command]
```
cog name, settings and commands should have this change too